include/can/core/messages.hpp

```diff
@@ -113,18 +113,18 @@ struct GetStatusResponse : BaseMessage<MessageId::get_status_response> {
 };
 
 struct MoveRequest : BaseMessage<MessageId::move_request> {
-    uint32_t steps;
+    int32_t target_position;
 
     template <bit_utils::ByteIterator Input, typename Limit>
     static auto parse(Input body, Limit limit) -> MoveRequest {
-        uint32_t steps = 0;
-        body = bit_utils::bytes_to_int(body, limit, steps);
-        return MoveRequest{{}, steps};
+        int32_t target_position = 0;
```

Comment:
if we're going to keep positions such that one axis extent is always 0 and the other is positive, let's keep absolute positions as unsigned types like `uint32_t`